doc/usage.md

```diff
@@ -385,34 +402,45 @@ None
 `--device virtio-input,pointing`
 
 
-## RESTful Service
+## RESTful API
+
+To interact with the RESTful API, append a valid scheme to your base command: `--restful-uri tcp://localhost:8081`.
 
 ### Get VM state
 
-Used to obtain the state of the virtual machine that is being run by vfkit.
+Obtain the state of the virtual machine that is being run by vfkit.
+
+Request:
+```HTTP
+GET /vm/state
+```
 
-GET `/vm/state`
-Response: { "state": string, "canStart": bool, "canPause": bool, "canResume": bool, "canStop": bool, "canHardStop": bool }
+Response:
+`{ "state": string, "canStart": bool, "canPause": bool, "canResume": bool, "canStop": bool, "canHardStop": bool }`
 
-> `canHardStop` is only supported on macOS 12 and newer, false will always be returned on older versions.
+`canHardStop` is only supported on macOS 12 and newer, false will always be returned on older versions.
 
 ### Change VM State
 
-Change the state of the virtual machine. Valid states are:
+Change the state of the virtual machine. Valid state values are:
 * HardStop
 * Pause
 * Resume
 * Stop
 
-POST `/vm/state` { "state": "new value"}
-
-Response: http 200
+```HTTP
+POST /vm/state { "state": "new value"}
+```
+Response: HTTP 200
 
 ### Inspect VM
 
 Get description of the virtual machine
 
-GET `/vm/inspect`
+```HTTP
+GET /vm/inspect
+```
+
```

Comment:
Thanks for all the typo/formatting/... fixes, I've split them to their own commit.